go.mod

```diff
@@ -1,8 +1,14 @@
 module github.com/posthog/posthog-go
 
-go 1.15
+go 1.18
 
 require (
 	github.com/google/uuid v1.3.0
 	github.com/urfave/cli v1.22.5
 )
+
+require (
+	github.com/cpuguy83/go-md2man/v2 v2.0.0-20190314233015-f79a8a8ca69d // indirect
```

Comment:
these are a bit sus, what are they used for? 🤔

Reply:
I just ran `go mod tidy`. This seems to be a dependency of [`github.com/urfave/cli`](https://github.com/urfave/cli/blob/v1.22.5/go.mod#L7).